.env.sample

```diff
@@ -21,8 +21,10 @@ ESTORE_URL=
 ESTORE_CRON_MANAGER_SCHEDULE="0 */1 * * * *"
 
 # MONGO DB
-MONGO_INITDB_DATABASE=
-MONGODB_URI=
+MONGO_INITDB_DATABASE=dtfs-submissions
+MONGO_INITDB_ROOT_USERNAME=root
+MONGO_INITDB_ROOT_PASSWORD=r00t
+MONGODB_URI=mongodb://root:r00t@dtfs-mongo:27017/?authMechanism=DEFAULT
```

Comment:
Should we have values here, most of the other examples have empty values?

Reply:
Apologies they were added to `MONGODB_URI_QA`.